### PR TITLE
[WIP]Add the missing test for enqueueCStorReplica

### PR DESCRIPTION
**Pull request by farhaanbukhsh**
This commit add the initail work in progress for the tests,
this is done in order to get review.
fixes: https://github.com/openebs/openebs/issues/1919

Signed-off-by: Farhaan Bukhsh <farhaan.bukhsh@gmail.com>

### CODE REVIEW

**Comment by farhaanbukhsh** [comment]
Hey it's a work in progress would be nice if I can get a review on this, am I doing something wrong because the queue doesn't seem to have any element.

**Comment by Akash4927** [comment]
Hi @farhaanbukhsh, can you please provide the issue link in the description?

**Comment by sonasingh46** [comment]
@farhaanbukhsh 
Thanks.
Please add issue link to get it reviewed.

**Comment by farhaanbukhsh** [comment]
@Akash4927 @sonasingh46 https://github.com/openebs/openebs/issues/1919

here it is

**Comment by Akash4927** [comment]
@farhaanbukhsh if you are done with the changes, please remove WIP from PR heading

**Comment by farhaanbukhsh** [comment]
Does it look good?

**Comment by farhaanbukhsh** [comment]
I actually wanted a review of you guys on this

**Comment by sonasingh46** [comment]
> @Akash4927 @sonasingh46 [openebs/openebs#1919](https://github.com/openebs/openebs/issues/1919)
> 
> here it is

It would be great if you put issue link in PR description in format : 
(fixes <issue_link>)
@farhaanbukhsh

File: cmd/cstor-pool-mgmt/controller/replica-controller/new_replica_controller_test.go [hunk 1]
```diff
@@ -48,3 +52,34 @@ func TestNewCStorVolumeReplicaController(t *testing.T) {
...
+				"cstorpool.openebs.io/name": "cstor-123abc",
+			},
+		},
+		Spec: apis.CStorVolumeReplicaSpec{
+			TargetIP: "127.0.0.1",
+			Capacity: "100MB",
+		},
+	}
+	q := common.QueueLoad{}
+	volumeReplicaController.enqueueCStorReplica(&test, q)
```
Comment (sonasingh46) [inline cmd/cstor-pool-mgmt/controller/replica-controller/new_replica_controller_test.go]: Good effort @farhaanbukhsh 
Ideally speaking we should not test this function as it in turn uses all the inbuilt library functions.
Additionally, pushing into queue won't reflect size immediately as it is a rate limited and non blocking call.
I cannot think of any cases where we might need to test the function.

Comment (farhaanbukhsh) [reply cmd/cstor-pool-mgmt/controller/replica-controller/new_replica_controller_test.go]: ahh okay so what should I be testing :disappointed: ?

**Comment by sonasingh46** [comment]
Will need to see the testing scope.
As of now,closing this PR.

**Pull request closed without merging**